pages/narratives/_book/_chapter/_page.vue

```diff
@@ -42,10 +42,11 @@ export default {
         // No Array.prototype function, so we can break the loop
         for (const entry of entries) {
           if (entry.isIntersecting) {
+            const { base = '' } = this.$router.options
             const book = this.$route.params.book
             const chapter = this.$route.params.chapter
             const page = entry.target.id
-            const path = `/narratives/${book}/${chapter}/${page}`
+            const path = `${base}narratives/${book}/${chapter}/${page}`
```

Comment:
shouldn't this read ${base}/narratives/${book}... etc.

Reply:
nope, base by default is `/`. if you add a slash it will become `//narratives/...`, which will be interpreted as `http://narratives/...